woostarter/styles/block/post-terms-with-background.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/theme.json",
+    "version": 3,
+    "slug": "post-terms-with-background",
+    "title": "With Background",
```

Comment:
I'd name this something like "button" or "badge" or something else so it doesn't get truncated in the label